src/SprykerEco/Zed/PunchoutCatalogs/Communication/Form/PunchoutCatalogConnectionForm.php

```diff
@@ -274,6 +278,7 @@ protected function addActiveDependentFieldSubFormToConnectionForm(FormInterface
             [
                 'inherit_data' => true,
                 'label' => false,
+                'validation_groups' => null,
```

Comment:
Isn't it a default value? Can you remove it?

Reply:
Nope, by default it's `disabled`, this method overwrites options for field which was added earlier

Reply:
👍